site/templates/guides.php

```diff
@@ -2,8 +2,13 @@
 
 <div class="mb-36">
   <h1 class="h1 mb-12">Guide</h1>
-  <ul class="guides auto-fill auto-rows-fr mb-12" style="--min: 16rem; --gap: var(--spacing-12)">
-    <?php foreach ($page->children()->listed() as $guide): ?>
+	<?php $categories = $page->children()->listed()->pluck('category', ',', true);
```

Comment:
I'd move this to the controller

Reply:
Maybe we could also use `groupBy()` instead.

Reply:
We should, I guess I didn't because I originally had some guides that didn't have a category yet, which usually leads to an error.